superset/reports/schemas.py

```diff
@@ -146,6 +146,12 @@ class ReportSchedulePostSchema(Schema):
         allow_none=True,
         required=False,
     )
+    email_subject = fields.String(
+        description=description_description,
+        allow_none=True,
+        required=False,
+        example="[Report]  Report name: Dashboard or chart name",
```

Comment:
I think you want to move `example` and description into a metadata dictionary. You can also define a new description variable for the email_subject.

Reply:
Modified